cabal-ghc901.project

```diff
@@ -41,22 +41,25 @@ index-state: 2021-12-29T12:30:08Z
 
 constraints:
   -- These plugins don't work on GHC9 yet
-  haskell-language-server +ignore-plugins-ghc-bounds -brittany -stylishhaskell -tactic,
+  -- Add a plugin needs remove the -flag but also update ghc bounds in hls.cabal
+  haskell-language-server +ignore-plugins-ghc-bounds -class -stylishhaskell -tactic,
```

Comment:
Is this re-disabling of class intended?

Reply:
hmm, not sure if i understand it, but just in case, `-flag` sets the flag value to false, so removing `-brittany` make the build use the default value (true)

Reply:
no, no, good catch, thanks